x/crosschain/keeper/cctx_gateway_zevm.go

```diff
@@ -28,7 +28,7 @@ func (c CCTXGatewayZEVM) InitiateOutbound(
 
 	if err != nil && !isContractReverted {
 		// exceptional case; internal error; should abort CCTX
-		config.CCTX.SetAbort(err.Error())
+		config.CCTX.SetAbort("Aborted while trying to handle EVM deposit", err.Error())
```

Comment:
We should add more details:
"error during deposit that is not smart contract revert"